scripts/load-backup.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+set -e
+
+# Check arguments
+if [ "$#" -ne 1 ]; then
+  echo "usage: $0 osrd.backup"
+  exit 1
+fi
+
+# Check sha1 is matching
+echo Checking backup integrity...
+BACKUP_PATH="$1"
+EXPECTED_SHA1=$(basename -s ".backup" "$BACKUP_PATH")
+EXPECTED_SHA1=${EXPECTED_SHA1##*sha1_}
+CURRENT_SHA1=$(cut -d' ' -f1 <<< $(sha1sum "$BACKUP_PATH"))
+if [ "$EXPECTED_SHA1" == "$CURRENT_SHA1" ]; then
+  echo "  ✔ The backup is valid"
+else
```

Comment:
It does not work if the file was renamed after download.
Failing in this case is ok, but we should detect that the file was renamed and print something